client/my-sites/plans/current-plan/controller.jsx

```diff
@@ -30,6 +30,12 @@ export function currentPlan( context, next ) {
 
 	const requestThankYou = context.query.hasOwnProperty( 'thank-you' );
 
+	// On plans grid page we might've scrolled down.
+	// Ensure we're scrolled to top when "thank you" modal is shown.
+	if ( requestThankYou ) {
```

Comment:
- I don't think the modal version is special (discussed via another channel)
- It's always good to guard `window` usage.

```suggestion
	if ( typeof window !== 'undefined' ) {
```

Reply:
Thanks, updated!

Switched to generic approach that covers the whole "My plan" section.

Spotted the same at "Plans" section:

https://github.com/Automattic/wp-calypso/blob/140b6f9d7b49def5961586bc7e955b554d002070/client/my-sites/plans/main.jsx#L47-L54